Tools/glib/dependencies/apt

```diff
@@ -36,50 +36,49 @@ PACKAGES=(
     cmake
     flite1-dev
     gawk
-    $(aptIfExists gi-docgen)
     gperf
     gtk-doc-tools
     intltool
     itstool
     libasound2-dev
     libatk1.0-dev
-    $(aptIfExists libavif-dev)
     libdrm-dev
-    $(aptIfExists libenchant-2-dev)
+    libenchant-2-dev
     libepoxy-dev
     libevent-dev
     libfile-copy-recursive-perl
-    libinput-dev
-    $(aptIfExists libjxl-dev)
-    $(aptIfElse libgcrypt20-dev libgcrypt11-dev)
+    libgcrypt20-dev
     libgstreamer1.0-dev
     libgstreamer-plugins-bad1.0-dev
     libgstreamer-plugins-base1.0-dev
+    libinput-dev
     libjpeg-dev
     libkate-dev
     liblcms2-dev
-    $(aptIfExists libmanette-0.2-dev)
+    libmanette-0.2-dev
     libnghttp2-dev
     libopenjp2-7-dev
     libpng-dev
     libseccomp-dev
-    $(aptIfExists libsoup-3.0-dev)
     libsqlite3-dev
-    $(aptIfExists libsysprof-capture-4-dev)
     libsystemd-dev
     libtasn1-6-dev
     libtool
     libwayland-dev
     libwebp-dev
     libwoff-dev
-    $(aptIfExists libwpe-1.0-dev)
-    $(aptIfExists libwpebackend-fdo-1.0-dev)
     libxml2-utils
     libxslt1-dev
     mesa-common-dev
     ninja-build
     patch
+    patchelf
```

Comment:
I highly doubt WebKit itself depends on `patchelf`. What would be this used for?

Reply:
That's correct, `patchelf` is not required to build WebKitGTK or WPE, but there are several scripts in `Tools/Scripts`, such as `Tools/Scripts/bundle-binary` and `Tools/Scripts/generate-bundle`, that use `patchelf`.

In addition, `patchelf` is also bundled in the FlatpakSDK (`Tools/buildstream/elements/sdk/patchelf.bst`), so it makes sense to me that it's part of the list of OS packages.